app/models/medical_history.rb

```diff
@@ -0,0 +1,7 @@
+class MedicalHistory < ApplicationRecord
+  include Mergeable
+  belongs_to :patient, optional: true
```

Comment:
Is this optional to allow async record creation? I mean the patient association _isn't_ optional.

Reply:
Yes, patient association is optional because of weak references between patient and medical history. 
Patients and Medical histories can be synced independent of each other.